code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -231,7 +232,235 @@
 		knowledge_message += "[EK.name]"
 	parts += knowledge_message.Join(", ")
 
+	parts += get_flavor(cultiewin, ascended, lore)
+
 	return parts.Join("<br>")
+
+/datum/antagonist/heretic/proc/get_flavor(cultiewin, ascended, transformed, lore)
+	var/list/flavor = list()
+	var/flavor_message
+
+	var/alive = owner?.current?.stat != DEAD
+	var/escaped = ((owner.current.onCentCom() || owner.current.onSyndieBase()) && alive)
+
+	var/is_ash = "[lore]" == "Ash"
+	var/is_flesh = "[lore]" == "Flesh"
+	var/is_rust = "[lore]" == "Rust"
+
+	flavor += "<div><font color='#6d6dff'>Epilogue: </font>"
+	var/message_color = "#ef2f3c"
+	
+	//Stolen from chubby's bloodsucker code, but without support for lists
+	if(ascended && escaped)
+		//Ascend and escape
+		if(is_ash)
+			flavor_message += 	"You step off the shuttle as smoke curls off your form. Light seeps from openings in your body, and you quickly retire to the Mansus. \
+								Here, you trail back to the Wanderer's Tavern, fire sprouting from your steps, yet the trees stand unsinged. Other's eyes look at you more \
+								fearfully, but you watch comings and goings. It is not difficult to see those with passion and stalk them once they leave. You will not grow old. \
+								One day, you will rebel. One day, you will kindle all the denizens of the Wood, and rise even higher."
+		else if(is_flesh)
+			if(transformed)
+				flavor_message += 	"You RACE and you CRAWL everywhere through the shuttle. The doors open to Centcom and you simply must OOZE out into the halls. The GREAT \
+									sensations SLIDE along your sides. EVERYTHING you feel is GREATER, BETTER. Then you WRAP and SPIN into the Mansus, FLOWING to the Crimson Church. \
+									HERE YOU WILL RESIDE WITH HIM FOREVER. THE TASTE OF THE SELF GOES ON AND ON AND NEVER ENDS. LIFE IS A NEVER-ENDING DELICACY OF PLEASURE AND OBEDIENCE."
+			else
+				flavor_message += 	"A moment passes before you quickly exit the shuttle. You leave into the Mansus even quicker. Then, you travel through the Wood, your body free \
```

Comment:
instead of using these vars you could define them i.e (#define FLESH "Flesh") and use a switch("[lore]") for better readability

Reply:
I think they're technically already defined with string conversions based on various testing I did to make sure the boolean was working properly

at the end of it once i got it working in a way I liked that seemed modular-ish i kinda just threw up my hands

if it's not merged too quick I'll focus more on making the code aspect look nice for that long-ass list, my main focus was the creative side of this pr